backend/endpoints/messages.py

```diff
@@ -77,3 +76,98 @@ async def send_message(
         status_code=status.HTTP_424_FAILED_DEPENDENCY,
         detail={"Message not sent": response},
     )
+
+
+@router.put(
```

Comment:
`status_code` is missing inside `@router.put()`.
@pauline-banye

Reply:
@zxenonx it's in responses. I didn't see the need to duplicate it.